spec/rubocop/cop/style/mixin_grouping_spec.rb

```diff
@@ -45,14 +45,17 @@
       let(:message) { 'Put `include` mixins in separate statements.' }
 
       context 'with several mixins in one call' do
-        it_behaves_like 'code with offense', <<-END.strip_indent
-          class Foo
-            include Bar, Qux
-          end
-        END
+        it_behaves_like 'code with offense',
+                        ['class Foo',
+                         '  include Bar, Qux',
```

Comment:
As per [this comment](https://github.com/bbatsov/rubocop/issues/3936#issuecomment-280255352), mixins are applied last-to-first when using this style, so the auto-correct should be:

```ruby
include Qux
include Bar
```

This surprised me a bit at first. 🙂